lib/render/Visualizer.cpp

```diff
@@ -998,11 +998,13 @@ int Visualizer:: captureImage(string filename)
     else // write png files
     {
 		FILE* test = fopen((const char*)filename.c_str(), "wb");
-        fclose( test );
-		if (!test) {
+		if (!test) 
```

Comment:
`FileOperationChecker` should be used here to simplify and modulate code.

Reply:
FileOperationChecker is in the GUI, and the Visualzier is in the render library.  Can't use it yet!

Reply:
that's true...